integration-tests/storybook-for-ember/package.json

```diff
@@ -15,18 +15,18 @@
   "devDependencies": {
     "@babel/core": "^7.1.2",
     "@percy/percy-storybook": "^3.0.0",
-    "@storybook/addon-a11y": "5.0.0",
-    "@storybook/addon-actions": "5.0.0",
-    "@storybook/addon-backgrounds": "5.0.0",
-    "@storybook/addon-centered": "5.0.0",
-    "@storybook/addon-knobs": "5.0.0",
-    "@storybook/addon-links": "5.0.0",
-    "@storybook/addon-notes": "5.0.0",
-    "@storybook/addon-options": "5.0.0",
-    "@storybook/addon-storysource": "5.0.0",
-    "@storybook/addon-viewport": "5.0.0",
-    "@storybook/addons": "5.0.0",
-    "@storybook/ember": "5.0.0",
+    "@storybook/addon-a11y": "v5.1.0-alpha.22",
```

Comment:
If you want to pin to the latest alpha, you can change these to `v5.1.0-alpha.0`

Reply:
Thanks.  I'll give that some thought.